.gitignore

```diff
@@ -1,42 +1,178 @@
-# Python compiled files
-*.py[ocd]
+# Standard gitignore list
 
-# C extensions
-*.so
```

Comment:
I think we should keep this

Reply:
Its still there, just re-organized a bit